oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -81,7 +81,7 @@ core:
     gRPCHost: ${SW_CORE_GRPC_HOST:0.0.0.0}
     gRPCPort: ${SW_CORE_GRPC_PORT:11800}
     maxConcurrentCallsPerConnection: ${SW_CORE_GRPC_MAX_CONCURRENT_CALL:0}
-    maxMessageSize: ${SW_CORE_GRPC_MAX_MESSAGE_SIZE:0}
+    maxMessageSize: ${SW_CORE_GRPC_MAX_MESSAGE_SIZE:52428800} #50MB
```

Comment:
Could you check sharing server config too?

Reply:
increased